### DISCUSSION TITLE
How to get relation names for AIFB used in RGCN

### DISCUSSION THREAD

**Discussion by YijianLiu** [pyg-team/pytorch_geometric#5689] [Q&A]
AIFB is Heterograph, but for the class entities to get AIFB graph, we just get relation type id.
But I really need to get relation type name for AIFB to know which edge is inverse, really need your help!

**Comment by rusty1s** [comment]
Can you take a look at the `raw` data? This information should be present there. Other than that, the reverse relation is always given by `rev_rel_type = rel_type + 1`, i.e., it is the direct next integer.
